namecomplete/candy.js

```diff
@@ -81,15 +81,7 @@ CandyShop.NameComplete = (function(self, Candy, $) {
 
 				});
 
-				// if we only have one match, no need to show the picker, just replace it
-				// else show the picker of the name matches
-				if (matches.length === 1) {
-					self.replaceName(matches[0]);
-					// Since the name will be autocompleted, throw away the last character
-					e.preventDefault();
-				} else if (matches.length > 1) {
-					self.showPicker(matches, field);
-				}
+				self.showPicker(matches, field);
```

Comment:
for non-english nickname, we use IME to input.
for example a user nickname `张三`, which will type as `zhangsan` in pinyin IME.
and there may be another user nickname as `zh99998`.
if I want to mention `张三`， when I Input `@zh`, it will auto select `@zh99998`.